.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false  # default: true
+contact_links:
+- name: Security bug report
+  url: https://docs.ansible.com/ansible-core/devel/community/reporting_bugs_and_features.html?utm_medium=github&utm_source=issue_template_chooser
```

Comment:
@gundalow originally I was trying to have different utm params for links from different places in case somebody tracking clicks would want to know sources. So you may want to add something like `_community_general`. FYI

Reply:
Good spot, thanks.
I've used `issue_template_chooser_ansible_collections` as 
1. These will move to `.github` repo soon
2. I don't want to have to maintain different files